go/libraries/doltcore/sqle/schema_table.go

```diff
@@ -15,65 +15,174 @@
 package sqle
 
 import (
+	"fmt"
+	"io"
+
 	"github.com/dolthub/go-mysql-server/sql"
 
 	"github.com/dolthub/dolt/go/libraries/doltcore/doltdb"
+	"github.com/dolthub/dolt/go/libraries/doltcore/row"
+	"github.com/dolthub/dolt/go/store/types"
 )
 
 // The fixed schema for the `dolt_schemas` table.
 func SchemasTableSchema() sql.Schema {
```

Comment:
We need to generate this schema as a Dolt `schema.Schema` in order to hard-code its tags. Currently it's are generated via the deterministic tag method, which is not stable when modifying the schema. See the [query catalog table](https://github.com/dolthub/dolt/blob/4f20d5f78c80885bc0a26a5a51d85cee6b5c14d7/go/libraries/doltcore/sqle/query_catalog_table.go#L29) for an example.